cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -202,6 +203,12 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "whether to validate integrity of image bootstrap",
 			Destination: &args.ValidateSignature,
 		},
+		&cli.StringFlag{
+			Name:        "kubeconfig-path",
+			Value:       "",
```

Comment:
Maybe provide a default path, like `$KUBECONFIG` or `~/.kube/config`.

Reply:
if path is empty，NewDefaultClientConfigLoadingRules will default to use ~/.kube/config or $KUBECONFIG